### PR TITLE
Feature/add banner for vote tab

### PR DESCRIPTION
**Pull request by svojsu**
TinderGov banner added for Gov tab

### CODE REVIEW

File: novawallet/Modules/Vote/Governance/TinderGov/TinderGovViewModelFactory.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+protocol TinderGovViewModelFactoryProtocol {
+    func createTinderGovReferendumsSection(
+        for referendums: [ReferendumLocal]?,
+        locale: Locale
+    ) -> ReferendumsSection?
+}
+
+struct TinderGovViewModelFactory: TinderGovViewModelFactoryProtocol {
+    func createTinderGovReferendumsSection(
+        for referendums: [ReferendumLocal]?,
```
Comment (ERussel) [inline novawallet/Modules/Vote/Governance/TinderGov/TinderGovViewModelFactory.swift:12]: Did you add votes in the next PR as we discussed?

**Review by ERussel — APPROVED** [review]

**Review by ERussel — COMMENTED** [review]
We need to fix the style of the banner. Currently see the following issues:
1) Background looks to dark, on the mockups it is blueish
2) "x referendums" corner radius smaller on mockups
3) there is a border on the mockups but can't see it on the screenshot

File: novawallet/Modules/Vote/Governance/TinderGov/TinderGovViewModelFactory.swift (hunk 1 above)
Comment (svojsu) [reply novawallet/Modules/Vote/Governance/TinderGov/TinderGovViewModelFactory.swift:12]: yep, I did

**Comment by svojsu** [comment]
> We need to fix the style of the banner. Currently see the following issues:
> 
> 1. Background looks to dark, on the mockups it is blueish
I don't see it's blueish on the mockups. Do you mean gradient smoothness may be?
> 2. "x referendums" corner radius smaller on mockups
The corner radius is wrong, thanks!
> 3. there is a border on the mockups but can't see it on the screenshot
there is a border on the screenshot. It's 1p, same as on the mockups. I could do it thicker so it will be more noticable

**Comment by svojsu** [comment]
upd

**Pull request merged**